invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/components/DisplayVerifiedCommunity.js

```diff
@@ -0,0 +1,53 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2024 CERN.
+//
+// Invenio App RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import React from "react";
+import PropTypes from "prop-types";
+import { Label, Image } from "semantic-ui-react";
+
+export const DisplayVerifiedCommunity = ({ communities }) => {
+  const displayCommunity = (parentCommunities) => {
+    const brandedCommunityId = parentCommunities?.default;
+    const communities = parentCommunities?.entries;
+    let displayCommunity = null;
+
+    if (communities) {
+      communities.map((community) => {
+        if (community.id === brandedCommunityId && community?.theme) {
+          displayCommunity = community;
+        }
+        return displayCommunity;
+      });
+    }
+
+    return displayCommunity;
+  };
+
+  const verifiedCommunity = displayCommunity(communities);
+  return (
+    <>
+      {verifiedCommunity && (
+        <Label
+          as="a"
+          href={`/communities/${verifiedCommunity.id}`}
+          style={{ backgroundColor: verifiedCommunity?.theme?.style?.primaryColor }}
```

Comment:
I think that should translate to a className rather that inline style if the community is verified